spec/rubocop/cop/style/alias_spec.rb

```diff
@@ -106,6 +106,51 @@ module M
       RUBY
     end
 
+    it 'registers an offense for alias in a def' do
+      expect_offense(<<~RUBY)
+        def foo
+          alias :ala :bala
+          ^^^^^ Use `alias_method` instead of `alias`.
+        end
+      RUBY
+
+      expect_correction(<<~RUBY)
+        def foo
+          alias_method :ala, :bala
+        end
+      RUBY
```

Comment:
This looks strange and like a bug. I think `NoMethodError` is not expected. WDYT?

```ruby
def foo
  alias :to_string :to_s
end

foo
p 42.to_string # => "#<Integer:0x0000000000000055>"

def bar
  alias_method :to_str, :to_s # => undefined method `alias_method' for main:Object (NoMethodError)
end

bar
p 42.to_str
```

Reply:
@ydah ping :-)

Reply:
Oops, I hadn't noticed. Thank you very much.
Indeed it is. It would seem that these should be corrected. Is there any problem to correct them in this PR? (In that case I will change the title)

Reply:
Indeed it is. I think it is better to fix it. Can we include the bug fix in this PR?

Reply:
Yes. This wrong behavior spec cannot be merged. Please update this PR.